### PR TITLE
[Merged by Bors] - chore: remove dependency on `activation/wire` from tests outside `activation`

### PR DESCRIPTION
**Pull request by fasmat**
## Motivation

Most packages should be unaware of wire types and not depend on packages implementing them (including tests).
This removes the dependency on `activation/wire` from most other packages.

## Description

The actual bytes that are stored don't matter for most tests and even before the change didn't match the domain object thats persisted in some tests. This just removes the dependency and uses random bytes instead.

**NOTE:** The `checkpoint` package still uses wire types and in my opinion shouldn't. But to get rid of that dependency we would need to store `prevATX` and `posATX` in the DB so we don't have to decode them from the stored blob.

## Test Plan

- existing tests pass

## TODO

- [x] Explain motivation or link existing issue(s)
- [x] Test changes and document test plan
- [x] Update documentation as needed
- [x] Update [changelog](../CHANGELOG.md) as needed

### CODE REVIEW

File: api/grpcserver/v2alpha1/activation_test.go [hunk 1]
```diff
@@ -28,9 +28,8 @@ func TestActivationService_List(t *testing.T) {
 	activations := make([]types.ActivationTx, 100)
 	for i := range activations {
 		atx := gen.Next()
-		vAtx := fixture.ToAtx(t, atx)
-		require.NoError(t, atxs.Add(db, vAtx, atx.Blob()))
-		activations[i] = *vAtx
+		require.NoError(t, atxs.Add(db, atx, types.AtxBlob{Blob: types.RandomBytes(100)}))
```
Comment (poszu) [inline api/grpcserver/v2alpha1/activation_test.go]: NIT: I think the blob could be empty:
```suggestion
		require.NoError(t, atxs.Add(db, atx, types.AtxBlob{}))
```

File: fetch/handler_test.go [hunk 2]
```diff
@@ -263,22 +261,15 @@ func TestHandleMeshHashReq(t *testing.T) {
 }
 
 func newAtx(t *testing.T, published types.EpochID) (*types.ActivationTx, types.AtxBlob) {
```
Comment (poszu) [inline fetch/handler_test.go]: NIT: How about returning just the ATX? The caller can create a fake blob if needed
```suggestion
func newAtx(t *testing.T, published types.EpochID) *types.ActivationTx {
```

**Review by poszu — APPROVED** [review]

**Comment by fasmat** [comment]
bors merge

**Comment by fasmat** [comment]
bors merge

**Review by poszu — APPROVED** [review]

**Comment by fasmat** [comment]
bors merge

**Comment by fasmat** [comment]
bors merge

**Pull request closed without merging**